### PR TITLE
Added mobile browser tests for android and iphone.

### PR DESCRIPTION
**Pull request by barribob**
We added back the test for mobile browserstack. Currently, browserstack is fairly limited, in that it only tests safari on ios and chrome on android. Also, currently, only iphone 6 and below connects to our server, and we cannot figure out why.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -358,8 +358,10 @@ tbody>td>.ordered-id {
     section {
         bottom: 130px;
         height: calc(100vh - 260px);
+        margin-top: 50px;
         overflow-x: hidden;
         padding-bottom: 60px;
+        top: 0px;
```
Comment (madison-codes) [inline css/styles.css]: You shouldn't need the px here -  should just be `top: 0;`.

File: nightwatch/tests/completed/getting-started-complete-test.js [hunk 2]
```diff
@@ -12,7 +12,6 @@ module.exports = {
         // Entering 'Set phasers to stun', clicking encrypt, and checking that the Encrypt table and log message appears
 
         Page.encryptMessage('Set phasers to stun');
-        Page.expectLog(`Encrypting plaintext 'Set phasers to stun'`);
```
Comment (madison-codes) [inline nightwatch/tests/completed/getting-started-complete-test.js:15]: Why is this removed?

**Review by madison-codes — APPROVED** [review]
Let's review tomorrow to make sure I understand what the current issues in Safari for iPhone are.

Otherwise just a couple of minor changes to make and then, barring I understand everything - let's merge this.

File: css/styles.css (hunk 1 above)
Comment (barribob) [reply css/styles.css]: Done

File: nightwatch/tests/completed/getting-started-complete-test.js (hunk 2 above)
Comment (barribob) [reply nightwatch/tests/completed/getting-started-complete-test.js:15]: There is some lag time between when the log appears for a phone for some reason, and because of that, we can't do an instant test for this. We decided that trying to make this test work is not worth the work required. As an alternate solution, we could have the expectLog test for the entire log, but that would not be as specific and could result in future tests passing when they actually failed. We can discuss more if you think this test is essential, as Ryan might have more insight into this.

**Comment by madison-codes** [comment]
@rylo5688 @miyo6032 - your code comments should typically come with the commit that addresses the initial review that your code comments are referencing.

**Comment by barribob** [comment]
We resolved the iPhone connectivity issue with Browserstack, but in order for iPhone to work, the localhost must be opened up to the public, which is not secure, so we are not going to add it in the pull request. It’s really an issue with Browserstack and they said they are in the process of fixing it. Details are in the google doc.

File: nightwatch/tests/completed/getting-started-complete-test.js (hunk 2 above)
Comment (ernieturner) [reply nightwatch/tests/completed/getting-started-complete-test.js:15]: Could you use the `this.waitForElementVisible` method to wait a bit and then do this assertion? That logic could all be completely wrapped up in the `expectLog` call possibly.
